loadgen/helpers.go

```diff
@@ -0,0 +1,34 @@
+package loadgen
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
+)
+
+func VisibilityCountIsEventually(
+	ctx context.Context,
+	client client.Client,
+	request *workflowservice.CountWorkflowExecutionsRequest,
+	expectedCount int,
+	waitAtMost time.Duration,
+) error {
+	deadline := time.Now().Add(waitAtMost)
+	for {
+		visibilityCount, err := client.CountWorkflow(ctx, request)
+		if err != nil {
+			return err
+		}
+		if visibilityCount.Count == int64(expectedCount) {
```

Comment:
```suggestion
		if visibilityCount.Count >= int64(expectedCount) {
```

Otherwise, this function may be less useful as a general helper

Reply:
The point here is for them to be exactly equal